src/App.js

```diff
@@ -2,13 +2,15 @@ import "./App.css";
 import Popup from "./components/Popup/Popup";
 import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
 import WelcomeView from "./components/WelcomeView/WelcomeView";
+import MainDashboard from "./components/MainDashboard/MainDashboard";
 
 const App = () => {
   return (
     <Router>
       <Switch>
         <Route path="/" exact component={WelcomeView} />
         <Route path="/popup" component={Popup} />
+        <Route path="/dashboard" component={MainDashboard}/>
```

Comment:
In the Routes components pass as `element={<Component/>}`  becasue of the newer react-router version